code/modules/vore/eating/belly_vr.dm

```diff
@@ -465,10 +465,16 @@
 			owner << "<span class='warning'>You feel your [name] start to cling onto its contents...</span>"
 			digest_mode = DM_ABSORB
 			return
-
-		else if(prob(digestchance) && digest_mode != DM_DIGEST) //Finally, let's see if it should run the digest chance.
+		
+		else if(prob(digestchance) && digest_mode != (DM_ITEMWEAK | DM_DIGEST)) //Finally, let's see if it should run the digest chance.
```

Comment:
Remember this affects player bellies too that have interactions turned on, but tbh I can't see it being a problem. (I need to ask around or do some sneaky VVing and see if anyone actually uses them)